pallets/rmrk-equip/src/functions.rs

```diff
@@ -235,9 +235,15 @@ where
 		ensure!(equipper_owner.0 == issuer, Error::<T>::PermissionError);
 
 		// Equipper must be direct parent of item
-		let equipper_owner = pallet_rmrk_core::Pallet::<T>::nfts(item_collection_id, item_nft_id)
-			.unwrap()
-			.owner;
+		let equipper_owner =
```

Comment:
We should change the naming bc `equipper_owner` get re-assigned from the `root_owner` to the `direct_owner` from line 231. I think we can change both lines to reflect better and avoid confusion.

[Line 231]
```
let (equipper_root_owner, _) = pallet_rmrk_core::Pallet::<T>::lookup_root_owner(
			equipper_collection_id,
			equipper_nft_id,
		)?;
		ensure!(equipper_root_owner == issuer, Error::<T>::PermissionError);
```

[Line 238]
```
let equipper_direct_owner = 
match pallet_rmrk_core::Pallet::<T>::nfts(item_collection_id, item_nft_id) {
				None => {
					// Item must exist (shouldn't happen here, already checked above)
					return Err(Error::<T>::ItemDoesntExist.into())
				},
				Some(nft) => nft.owner,
			};

		ensure!(
			equipper_direct_owner ==
				AccountIdOrCollectionNftTuple::CollectionAndNftTuple(
					equipper_collection_id,
					equipper_nft_id
				),
			Error::<T>::MustBeDirectParent
		);
  ```

Reply:
Thanks, updated